usaspending_api/awards/v1/views.py

```diff
@@ -182,3 +185,57 @@ def get_queryset(self):
         filtered_queryset = self.filter_records(self.request, queryset=queryset)
         ordered_queryset = self.order_records(self.request, queryset=filtered_queryset)
         return ordered_queryset
+
+
+class FABAFilter(filters.FilterSet):
+
+    class Meta:
+        model = FinancialAccountsByAwards
+        # You can create any relationships you want here using ORM notation
+        fields = ('piid', 'treasury_account__agency_id')
```

Comment:
Once this is converted to the more concise version using `filterset_fields` in the View class, can you see you can showcase some precision in being able to select _which kinds_ of filters ("lookups") are allowed using the dictionary syntax seen here: https://django-filter.readthedocs.io/en/latest/ref/filterset.html#declaring-filterable-fields?
```
fields = {
            'username': ['exact', 'contains'],
            'last_login': ['exact', 'year__gt'],
        }
```